### PR TITLE
Rdf suppport

### PR DESCRIPTION
**Pull request by The-Marvex**

### CODE REVIEW

File: org.bridgedb.tools.qc/src/org/bridgedb/tools/VoID/VoIDTool.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    /**
+     * this method creates RDF file from the derby database passed through the CLI Tool
+     * @throws SQLException
+     * @throws IDMapperException
+     */
+    public void createRDF() throws SQLException, IDMapperException {
+        Connection con = oldGdb.getConnection();
+        Statement st= con.createStatement();
+        for(DataSource ds : oldGdb.getCapabilities().getSupportedSrcDataSources()) {
+            String sql = "SELECT * from "+(ds.getFullName()).toUpperCase();
```
Comment (egonw) [inline org.bridgedb.tools.qc/src/org/bridgedb/tools/VoID/VoIDTool.java]: I think this belongs in `SimpleGdbImpl4`

**Comment by The-Marvex** [comment]
I'll fix it and update the pr.

**Comment by The-Marvex** [comment]
@egonw updated the PR

**Comment by egonw** [comment]
I'll try to pull them in on Wednesday! Please start writing up the work you did in one or two final blog posts, outlining in how the new functionality is meant to be used. Not necessarily in a lot of technical detail, but focusing on why people should adopt it.

**Pull request merged**